Godeps/Godeps.json

```diff
@@ -5,6 +5,7 @@
 	"Packages": [
 		"github.com/ugorji/go/codec/codecgen",
 		"github.com/onsi/ginkgo/ginkgo",
+		"github.com/square/go-jose",
```

Comment:
Ah, I've added this before adding the dummy code to force it in. I'm still waiting for it to report back on latests changes, will try removing it, if it comes back with a failure.